src/app/cheat-sheets/game-base/balancers/balancers.component.html

```diff
@@ -1,28 +1,17 @@
 <app-cheat-sheet [cheatSheet]="cheatSheet">
   <p>
-    <strong><a href="https://wiki.factorio.com/Balancer_mechanics">Balancers</a></strong>
-    are used to evenly distribute items over multiple belts, they are commonly used at ore patches or the
-    <a href="https://wiki.factorio.com/Tutorial:Main_bus#Split-off">Main Bus</a>.
+    Use <a href="https://wiki.factorio.com/Balancer_mechanics">balancers</a>
+    to distribute items over multiple belts, like at ore patches or on the
+    <a href="https://wiki.factorio.com/Tutorial:Main_bus#Split-off">main bus</a>.
   </p>
+  <p>There are four kinds of balancers:</p>
```

Comment:
Its not really 4 kinds though, there are more like properties of balancers rather than types